llm_bioinformatics_research/package.json

```diff
@@ -23,8 +23,6 @@
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "prestart": "node killport.js 3000 5000",
-    "killport": "node killport.js",
     "start": "concurrently \"npm run start-frontend\" \"npm run start-backend\"",
     "start-frontend": "react-scripts start",
```

Comment:
You did a good job of handling which port the backend server should be running on. But, I am having issues with starting the frontend server. It is dynamically set to port 3000, but if I have a process running on that port, it errors out. We will need to add functionality to check for the next available port if the default port is busy. I am including a screenshot of the error for your reference:

![image](https://github.com/user-attachments/assets/f097b6d6-f539-4958-a0f5-6d6e6bf31990)